.changeset/hip-hotels-type.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend': patch
+---
+
+Fixed an bug in the entity processing caching that would prevent entities that were emitted during processing to be restored after being overridden.
```

Comment:
Am I right in understanding that this will re-stitch too? At least to RUN the stitcher and then bail out with `unchanged`? But then again, probably mostly for Location entities themselves since most kinds don't emit children 🤔 So maybe it's benign enough of an event that we don't have to call it out here.

Reply:
yep expecting this to trigger stitch but hit the cache